### ISSUE TITLE
[FEATURE] Move host label parsing to clap

### ISSUE THREAD

**Issue by thomastaylor312** [wasmCloud/wasmCloud#757]
**Is your feature request related to a problem? Please describe.**

The current way we parse labels from environment variables is inside of library code rather than being at the level of command line arg definition with clap. This is an artifact from moving things over from the OTP host and makes it hard to follow what is going on with parsing data

**Describe the solution you'd like**

The label parsing code should be moved from https://github.com/wasmCloud/wasmCloud/blob/b07302fe0befb3c04cc4f3f4c3c58f8fd35be04a/crates/host/src/wasmbus/mod.rs#L1782 to the top level clap args in the `src` directory

**Comment by MeenuyD** [comment]
Hello @thomastaylor312 I would like to work on this issue can you please tell me where should change needs for this issue

**Comment by thomastaylor312** [comment]
Hey there @MeenuyD! Thanks for being willing to help. So the logic inside of this block: https://github.com/wasmCloud/wasmCloud/blob/b07302fe0befb3c04cc4f3f4c3c58f8fd35be04a/crates/host/src/wasmbus/mod.rs#L1770-L1793 should be handled somewhere right after here: https://github.com/wasmCloud/wasmCloud/blob/e4889ae8af1d404521ce0c823399e768f59733c8/src/main.rs#L345. If this is your first time contributing to a Rust project, please feel free to open a draft PR and we can help walk you though!

**Comment by MeenuyD** [comment]
Should I do the whole logic from the mod.rs to the main.rs

**Comment by MeenuyD** [comment]
Or we can create it as the function

**Comment by professorabhay** [comment]
hey @thomastaylor312, I tried to resolve the issue and raised the [PR](https://github.com/wasmCloud/wasmCloud/pull/783).
Please take a look and let me know any changes you want. I am newbie with rust and its first time to contributing to WASMCLOUD.
